runtime/ftplugin/qf_toc.vim

```diff
@@ -0,0 +1,23 @@
+function! s:setup_toc() abort
```

Comment:
It's how `ftplugin.vim` loads them:

```vim
exe 'runtime! ftplugin/' . name . '.vim ftplugin/' . name . '_*.vim ftplugin/' . name . '/*.vim'
```

I did this for `help` and `qf` because I was adding substantial code and not completely rewriting them.  It just felt right.  I can put it back if you're not into it.